### PR TITLE
Compiles on OS X

### PR DESCRIPTION
**Pull request by ricardoquesada**
compiles on Mac.

BTW, we need pure virtual functions for these kind of interfaces.

### CODE REVIEW

File: scripting/javascript/bindings/generated [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit f83f542e173bb82b281b617c31248f0ad8b01df7
+Subproject commit f85c95751a8a59846776789be30519e45b3911fd
```
Comment (dumganhar) [inline scripting/javascript/bindings/generated:1]: Why was the submodule changed?

**Comment by dumganhar** [comment]
> we need pure virtual functions for these kind of interfaces.

I totally agree. :)

**Comment by minggo** [comment]
CCAccelerometer is not interface, it is a class to register to receive message.

**Comment by dumganhar** [comment]
@minggo @ricardoquesada 

> CCAccelerometer is not interface, it is a class to register to receive message.

Yes, @minggo  is right. `CCAccelerometer` isn't a interface class.
But I think `CCAccelerometerDelegate` should be pure virtual class.

``` cpp
class CC_DLL CCAccelerometerDelegate
{
public:
    virtual void didAccelerate(CCAcceleration* pAccelerationValue)  = 0;  // Changing to pure virtual.
};
```

**Comment by minggo** [comment]
CCAccelerometerDelegate is removed.

**Comment by minggo** [comment]
Thanks. It is resolved in #2936.

**Pull request closed without merging**